cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database")

```

Comment:
It's a good practice to provide feedback to the user about what the system is doing. Good job on this.